modules/nf-core/chopper/tests/main.nf.test

```diff
@@ -19,6 +19,7 @@ nextflow_process {
                                 [id:'test_out' ], // meta map
                                 file(params.modules_testdata_base_path + 'genomics/sarscov2/nanopore/fastq/test.fastq.gz', checkIfExists: true)
                             ]
+                input[1] = file("https://github.com/wdecoster/nanolyse/raw/master/reference/lambda.fasta.gz", checkIfExists: true)
```

Comment:
Please use files in the nf-core test-datasets repo (modules beranch), we don't have control over the nanolyse repo so if the file gets moved or delted it will break it for us.

Reply:
IF you can find an 'official' version e.g. from NCBI we could upload that instead (adn document where we got it from)

Reply:
Sure, I just copied how the tests for the nanolyse module was done. I can upload the lambda genome to test-data modules branch, but I could also just use another viral fasta files available in test data, to avoid having having more data in the repo.